capstone_project/src/main/webapp/index.html

```diff
@@ -4,14 +4,39 @@
     <meta charset="UTF-8">
     <title>STEP Pod 155 Capstone</title>
     <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Raleway">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/css/materialize.min.css">
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/js/materialize.min.js"></script>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/js/bootstrap.min.js"></script>
   </head>
   <body>
-    <div id="content">
-      <h1>STEP Pod 155 Capstone</h1>
-      <p>This is the capstone project for STEP Pod 155.</p>
-      <p>Click <a href="pages/maps.html">here</a> to explore our maps feature.</p>
-      <p>Click <a href="pages/analyze.html">here</a> to check out our sentiment analysis feature.</p>
-      <p>Click <a href="pages/comments.jsp">here</a> to upload a post and view post submissions.</p>
-    </div>
+    <section>
+      <div id="header">
+        <div id="intro">
+          <h1><span class="typewrite">UniSocial</span></h1>
+          <hr class="hr-light">
+          <h6>
+            Built for students, by students.
+          </h6>
+          <br>
+          <div class="center">
+            <form action="pages/comments.jsp">
+                <input type="submit" class="button" value="CREATE A POST">
+            </form>
+            <form action="pages/maps.html">
+                <input type="submit" class="button" value="EXPLORE SCHOOLS">
+            </form>
+            <form action="pages/resources.html">
+                <input type="submit" class="button" value="CHECK OUT RESOURCES">
+            </form>
+          </div>
+        </div>
+      </div>
+    </section>
   </body>
 </html>
+
+
```

Comment:
nit: remove this extra blank line. I think there should just be one at the end, not two